engine/src/main/java/org/terasology/engine/rendering/opengl/FBO.java

```diff
@@ -728,6 +728,9 @@ public int height() {
          * @param height An integer representing the new height.
          */
         public void setDimensions(int width, int height) {
+            if(width == 0 && height ==0){
```

Comment:
Minor code quality fixes here, explained by the robot :-)

Should this just check for an _or_ rather than _and_ ? In the case we're dealing with _both_ dimensions are zero. But it seems like _either_ being zero would be a case where you'd disregard the instruction and return early.

Additionally, maybe it would be good if we did a `logger.warn(....)` to highlight that the set was called with invalid dimensions?